apps/wallet/src/ui/app/pages/welcome/index.tsx

```diff
@@ -15,63 +15,70 @@ import PageLayout from '_pages/layout';
-                                            weight="medium"
+                                <div className="mx-auto mt-7">
+                                    <div className="text-center">
+                                        <Heading
+                                            variant="heading2"
+                                            color="gray-90"
+                                            as="h1"
+                                            weight="bold"
                                         >
-                                            Buy, store, send and swap tokens
-                                        </Text>
+                                            Welcome to Sui Wallet
+                                        </Heading>
+                                        <div className="mt-3">
+                                            <Text
+                                                variant="p1"
+                                                color="steel-dark"
+                                                weight="medium"
+                                            >
+                                                Connecting you to the
+                                                decentralized web and Sui
+                                                network.
+                                            </Text>
+                                        </div>
                                     </div>
-                                    <div className="flex gap-2 mt-3 items-center">
-                                        <Icon
-                                            icon={SuiIcons.Checkmark}
-                                            className="text-success text-[8px]"
-                                        />
-                                        <Text
-                                            variant="body"
-                                            color="steel-dark"
-                                            weight="medium"
-                                        >
-                                            Explore blockchain apps
-                                        </Text>
+
+                                    <div className="mt-5 flex gap-2 flex-col">
+                                        {VALUE_PROP.map((value) => (
+                                            <div
+                                                key={value}
+                                                className="flex gap-2 items-center border bg-sui-light/40 border-sui/30 border-solid rounded-xl px-3 py-2"
+                                            >
+                                                <Icon
+                                                    icon={SuiIcons.Checkmark}
+                                                    className="text-white text-[6px] bg-steel rounded-full h-4 min-w-[16px] w-4 flex items-center justify-center"
```

Comment:
probably only `text-steel flex-none` will be necessary after using `@mysten/icons`

Reply:
Thanks for the comment @pchrysochoidis . It seems I still need to add height and width for the icon here though. It stays at 12px otherwise. Not 100% sure why it gets scaled down in this page. Please review again.